core/app/models/spree/order.rb

```diff
@@ -787,7 +787,7 @@ def after_cancel
     end
 
     def send_cancel_email
-      OrderMailer.cancel_email(self.id).deliver_later
+      OrderMailer.cancel_email(self).deliver_later
```

Comment:
@BenMorganIO yah the options I see are a) Find a way to make it backwards compatible, or b) Let people know they need to update their code (in the changelog or something?)